receiver/azuremonitorreceiver/config.go

```diff
@@ -20,6 +20,10 @@ var (
 	errMissingClientID       = errors.New(`ClientID" is not specified in config`)
 	errMissingClientSecret   = errors.New(`ClientSecret" is not specified in config`)
 	errMissingFedTokenFile   = errors.New(`FederatedTokenFile is not specified in config`)
+	errInvalidCloud          = errors.New(`Cloud" is invalid`)
+
+	azureCloud           = "AzureCloud"
```

Comment:
I am not sure these are the preferred names, but these are the names returned from running `az cloud list`. Other tools use different conventions (e.g. terraform).